src/utils/getKeys.ts

```diff
@@ -0,0 +1,24 @@
+// main
+import type { StringKeys } from '@transcend-io/type-utils';
```

Comment:
these are all exposed in type-utils, can we just import getStringKeys, and getKeys from there?